components/arm/arm.go

```diff
@@ -265,6 +272,12 @@ func (r *reconfigurableArm) Close(ctx context.Context) error {
 	return viamutils.TryClose(ctx, r.actual)
 }
 
+func (r *reconfigurableArm) IsMoving(ctx context.Context) (bool, error) {
```

Comment:
It's needed, unfortunately. Without adding "IsMoving" into GRPC (and thus, the main, non Local Interfaces) the rpc.Client that represents modular resources within the parent can't provide the function, and thus the subtype service can't build status for any of those types, and then the UI can't display cards for them. The other option would be to remove "IsMoving" from all the status stuff.

I would REALLY like to get rid of the LocalXXXXX variations of interfaces as much as possible, and internally, if something absolutely needs some other function, then do a specific cast then and there for it.

Reply:
But wouldn't that have been broken prior to this PR? Regarding the local interfaces, I think they may have been getting used incorrectly but they have a place for any kind of SDK, I believe

Reply:
So, prior to this, all resources in the resource manager were local objects WITH Local interfaces. So the "Status" function registered for the subtype services in question would just call IsMoving() (and other stuff.) Now any number of resources can be modular resources (which are grpc client objects) and thus the code breaks when trying to gather Status, because that client object couldn't support IsMoving().